### PR TITLE
Moddified style.less to fix Issue # 1334

### PR DESCRIPTION
**Pull request by JohnEJames**
I modified the style.less file by doing these things:
changed the background color to ghostwhite (trying to make it not blend in with a white background)
I increased the box shadow by 10 px instead of its original 2px (To have a shadow at the top)
Added a border 1px with the color gainsboro (To help avoid the bleeding effect)
and  when collapsed that there is no border  (So no border would show over the Remix button till hovered over)

Here is what it looks like with my changes
e3b121f2d0ec58c954c7656e4d2d093c

@humphd  If there is anything you would like me to change please let me know

### CODE REVIEW

**Comment by humphd** [comment]
Cool, I'll get @flukeout to weigh in on the styling changes.

**Comment by gideonthomas** [comment]
Fixes #1334

**Comment by flukeout** [comment]
Will check in today.

**Comment by flukeout** [comment]
Ok @JohnEJames thanks for submitting & getting this started! I reviewed the proposed style changes, and since this is an element present on every published project, I'd like to provide some specific guidance:

Expanded state...

image

* The border around the outside it 5px border with rgba(0,0,0,.15) as the color.
* You can use the ``background-clip: padding-box;`` rule on that element to make the border go to the "outside" of the box.
* For the .collapsed state, I added a transparent border of the same width.

Unexpanded + rollover
remix-rollout

Anyway, let me know if you're comfortable working towards what I've described, I realize it's very specific and design-heavy. If you can get most of the way there, I can help finalize the details too.

**Comment by JohnEJames** [comment]
@flukeout  I've made most of the changes you requested,  but I've been having trouble with the margin around the logo

5a40cf2c636f3144dbce5206e8121d42

It seems like it doesn't like having the padding at the bottom

**Comment by flukeout** [comment]
Getting there! If you update the PR with your latest changes I can help you out.

**Comment by JohnEJames** [comment]
Posted my changes @flukeout.  P.S I don't know if you get a notification for being in this conversation, If you do I'll stop tagging you

**Comment by flukeout** [comment]
@JohnEJames will look at this a little later today and advise. Oh yeah, please keep using the mention, it helps me keep track of where to pay attention to!

**Comment by gideonthomas** [comment]
@JohnEJames, I see some extra commits in here. Would you be able to try rebasing your branch onto master and pushing that up?

**Comment by flukeout** [comment]
Okay, here's a few changes that should get things looking good:

* Drop the ``box-shadow`` from the slide-out container
* Change it's height to ``75px``
* For the icon, give it a height of ``calc(100% - 10px)`` this sets it's height to 10px less than the container, but because it's LESS, you'll need to escape that rule like this ``~"calc(100% - 10px)"`` it wont' work otherwise
* Lastly, set the ``top`` and ``right`` properties of the ``.mouse-mode`` wrapper both to ``6px`` to give it a bit more breathing room.

Cheers!

**Comment by JohnEJames** [comment]
Ok, so I think I have it the way you wanted @flukeout, atm I'm trying to rebase it like you guys asked. Once I figured it all out, I'll commit it 
1618de4d22cd96e0d3040cefb8e9e34f

**Comment by flukeout** [comment]
Hey @JohnEJames - can you rebase and update your PR so it doesn't include all of the unrelated commits? Then I'll have another look and we'll get it merged!

**Comment by JohnEJames** [comment]
@flukeout  I think I rebased it properly, If not I'll try again

**Comment by flukeout** [comment]
Maybe @gideonthomas can help you sort this out. It still looks like a bunch of the commits are a part of the history of this pull request: https://github.com/mozilla/thimble.mozilla.org/pull/1712/commits

One option, which I've often used myself is to save the contents of the ``style.css`` file that you made. Then, create a brand new branch from the latest master branch, and apply your changes there. That should clean things up and might be easier than trying to rebase.

**Comment by JohnEJames** [comment]
Will I have to make a new pull request for that option? @flukeout

**Comment by humphd** [comment]
@JohnEJames let's try to fix this together.  Here's what you should do (assuming your remote to Mozilla's Thimble repo is named `upstream`, modify this if not):

```
git checkout master
git pull upstream master
git checkout Issue#1334
git rebase master
# fix any conflicts -- ask for help here if you have troubles with this step
git push Issue#1334 -f
```

**Comment by JohnEJames** [comment]
Ok I tried Exactly what you said @humphd  hopefully it worked.

**Comment by Pomax** [comment]
Somewhat counter intuitively, if you've addressed all the comments in a review you can dismiss the review as "addressed all issues mentioned in the review" (or something to that effect) and then in the "reviewers" option in the upper right you can *reselect* the same person to do a review. Github's UI is a little silly here: the "chat" icon next to someone's name means they left review comments, they are not marked as "needing to review" until you add them, or if they were already there, add them *again*, so the icon becomes the little yellow dot to indicate "they have been notified that they need to review but have not acted on that  yet".

(Not GitHub's best interface design)

File: public/resources/remix/style.less [hunk 1]
```diff
@@ -89,17 +89,19 @@
...
+    top: 6px !important;
     right: 6px !important;
-    height: 64px !important;
-    background: white !important;
+    height: 75px !important;
+    background: ghostwhite !important;
     padding: 11px 110px 11px 76px !important;
     box-sizing: border-box !important;
     font-family: "Fira Medium", "Helvetica Neue", sans-serif !important;
     box-shadow: 0 2px 2px rgba(0,0,0,.5) !important;
```
Comment (flukeout) [inline public/resources/remix/style.less]: Hey @JohnEJames - just get rid of this line (the box-shadow) and we'll be good to go!

Thanks for all of this work and for your patience. I know it's annoying to resolve all of these small styling issues commit by commit in this fashion, but I'm glad we're almost there.

**Comment by JohnEJames** [comment]
@flukeout  Okay, the line is removed. Does everything else look good? Also, I appreciate the help and being patient with me.

**Comment by flukeout** [comment]
We did it! Thanks so much for your work @JohnEJames

**Comment by humphd** [comment]
Nice, 💯 @JohnEJames

**Pull request merged**